tests/integration/components/new-signup/checkbox-test.js

```diff
@@ -0,0 +1,115 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// handler functions are creating some problems need to fix em
```

Comment:
Is this resolved?

Reply:
This is not my comment

Reply:
Done with changes can you please check my latest commit